wrappers/vue/README.md

```diff
@@ -67,7 +67,7 @@ The component will be available as `Handsontable.vue.HotTable`.
 
 ### Usage
 
-Use this data grid as you would any other component in your application. [Options](//handsontable.com/docs/Options.html) can be set as `HotTable` props.
+Use this data grid as you would any other component in your application. [Options](https:handsontable.com/docs/Options.html) can be set as `HotTable` props.
```

Comment:
```suggestion
Use this data grid as you would any other component in your application. [Options](https://handsontable.com/docs/api/options/) can be set as `HotTable` props.
```

Reply:
@budnix - done for all `Options.html` links: 67a6f3b